apps/ensadmin/src/components/ensadmin-version.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { Skeleton } from "@/components/ui/skeleton";
+import { ensAdminVersion } from "@/lib/env";
+import { Suspense, use } from "react";
+
+const versionPromise = ensAdminVersion();
```

Comment:
We use `async` to fetch the version from the `package.json` file inside `@/lib/env` so we need to wrap this in a promise and pass it around in React land as children/ReactElement so it can be suspended.